KubernetesPodOperator.rst

```diff
@@ -0,0 +1,955 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can you move it to https://github.com/apache/airflow/blob/master/docs/apache-airflow-providers-cncf-kubernetes/operators.rst?